src/prepare-styles/mixins.scss

```diff
@@ -6,6 +6,12 @@
   }
 }
 
+@mixin on-bigTablet {
+  @media (min-width: 768px) {
+    @content;
+  }
+}
```

Comment:
don't knowi f this is really necessary, because it makes code a little confusing, but let it stay here for now